lib/rmt/scc_sync.rb

```diff
@@ -40,6 +41,12 @@ def sync
       update_auth_token(item)
     end
 
+    @logger.info('Updating subscriptions')
+    data = scc_api_client.list_subscriptions
+    data.each do |item|
+      create_subscription(item)
+    end
+
     @logger.info('Done!')
   rescue SUSE::Connect::Api::InvalidCredentialsError
     raise CredentialsError, 'SCC credentials not valid.'
```

Comment:
Coverage shows me that this line is not tested at all.

Reply:
It's covered, are you running the full test suite?
That isn't covered, though:
```ruby
ProductPredecessorAssociation.create(product_id: product.id, predecessor_id: predecessor_id)
```

Reply:
I was running

```
$ bundle exec rspec spec
```

Reply:
Weird. Should be covered!